lib/rules/at-rule-allowed-list/index.js

```diff
@@ -47,6 +47,7 @@ const rule = (primary) => {
 				node: atRule,
 				result,
 				ruleName,
+				word: `@${atRule.name}`,
```

Comment:
[trivial] Reusing `name` seems good 👍🏼 

```suggestion
				word: `@${name}`,
```